hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip632/IsAuthorizedTest.java

```diff
@@ -663,6 +664,7 @@ record TestCase(long gasAmount, ResponseCodeEnum status) {}
                     }));
             final var hapiSpec = testCase.status() == SUCCESS
                     ? throughWhen.then(getTxnRecord(recordName)
+                            .logged()
```

Comment:
do we want to keep this in?  the output will be large but I guess no real harm.

Reply:
Removed it (but it was probably harmless as the side-effect on the logs isn't prohibitive).